### DISCUSSION TITLE
How to get enum values ?

### DISCUSSION THREAD

**Discussion by jgranduel** [duckdb/duckdb#3003] [Q&A]
Hi, 

while reading [The lord of enums](https://duckdb.org/2021/11/26/duck-enum.html), i wonder (and cannot find) a way to get values from an `enum`.

Could there be a `dot command` in the shell for instance, like `.schema` or `.tables` ? this might not be possible for SQLite shell compatibility though.

Thanks !

**Comment by Mytherin** [comment]
Perhaps you could use one of [these functions](https://github.com/duckdb/duckdb/issues/2741)?

**Reply by jgranduel** [reply]
Thanks! If I understand well, these function will be available in next release?

**Reply by Mytherin** [reply]
Yes, they should be.

**Reply by l1t1** [reply]
will next release available in the January?
